src/ethereum/shanghai/fork_types.py

```diff
@@ -173,18 +74,6 @@ def encode_account(raw_account_data: Account, storage_root: Bytes) -> Bytes:
     )
 
 
-@slotted_freezable
-@dataclass
-class Withdrawal:
-    """
-    Withdrawals that have been validated on the consensus layer.
-    """
-
-    index: U64
-    validator_index: U64
-    address: Address
-    amount: U256
-
```

Comment:
I think withdrawal is also a type of transaction and since I moved all the transactiontypes like LegacyTransaction and Transaction into separate module so it made more sense to keep withdrawal also within transaction.
In order to maintain a similar fashion .Thats it :)

Reply:
@SamWilsn  What's your suggestion on this because I am not sure about it. Its just that withdrawal here is different from normal withdrawal transaction that we think of in traditional finance so it would be better in fork_types ig.
if you say I will revert back the change that I did

Reply:
`Withdrawal` doesn't _seem_ like a transaction, and if there are going to be different types, yeah I agree that it'll probably deserve its own module in the future.

I'll move it back and start the tests again.